### ISSUE TITLE
quality of results on the Aiuniai website is significantly higher

### ISSUE THREAD

**Issue by theSha1do1w** [AiuniAI/Unique3D#88]
I observed that the quality of results on the aiuniai website is significantly higher than the model I deployed locally. May I ask if a different model was used or different parameters were applied?

**Comment by ResiduaLiu** [comment]
It is a different model

**Comment by josepmy** [comment]
@theSha1do1w Yes, there are probably improvements implemented that are not updated in the repository. But what they are for try to find a way to improve too? Multiview to 6 should be one.
